app/views/document_sets/_edit_privacy_form.html.slim

```diff
@@ -1,27 +1,37 @@
 tr
   th(colspan="2")
     h3.settings-label
-      =t('document_sets.edit_privacy.document_set_privacy', privacy: document_set.is_public ? t('document_sets.edit_privacy.public') : t('document_sets.edit_privacy.private')).html_safe
-    -if document_set.restricted
-      =button_tag name: 'document_set[is_public]', value: 'true', class: 'button inline-btn' do
-        = svg_symbol '#icon-unlock', class: 'icon'
-        span =t('document_sets.edit_privacy.make_document_set_public')
-    -else
-      =button_tag name: 'document_set[is_public]', value: 'false', class: 'button inline-btn' do
-        = svg_symbol '#icon-lock', class: 'icon'
-        span =t('document_sets.edit_privacy.make_document_set_private')
+      =t("document_sets.edit_privacy.document_set_privacy_#{document_set.visibility}").html_safe
 tr
   td(colspan="2")
     -if document_set.restricted
       p =t('document_sets.edit_privacy.document_set_restricted_description')
     -else
       p =t('document_sets.edit_privacy.document_set_unrestricted_description')
+tr
+  td(colspan="2")
+    =f.radio_button :visibility, :public, name: 'document_set[visibility]', checked: document_set.visibility_public?
+    =f.label 'document_set_public'
```

Comment:
For some reason clicking on the labels does not click the radio button, which is contrary to what I'd expect.